relay-event-normalization/src/normalize/span/ai.rs

```diff
@@ -47,43 +57,88 @@ pub fn extract_ai_measurements(span: &mut Span, ai_model_costs: &ModelCosts) {
         return;
     };
 
-    let total_tokens_used = measurements.get_value("ai_total_tokens_used");
-    let prompt_tokens_used = measurements.get_value("ai_prompt_tokens_used");
-    let completion_tokens_used = measurements.get_value("ai_completion_tokens_used");
-    if let Some(model_id) = span
-        .data
+    let data = span.data.get_or_insert_with(SpanData::default);
+
+    if let Some(ai_total_tokens_used) = measurements.get_value("ai_total_tokens_used") {
+        data.gen_ai_usage_total_tokens
+            .set_value(Value::F64(ai_total_tokens_used).into());
+    }
```

Comment:
This would override data, maybe something like this instead, assuming this wasn't on purpose?

```rust
    if data.gen_ai_usage_total_tokens.value().is_empty() {
        data.gen_ai_usage_total_tokens = measurements
            .get_value("ai_total_tokens_used")
            .map(Value::F64)
            .into();
    }
```